on_a() {
    cause_game_fn_error()
}
